libc/src/string/strcmp.c

```diff
@@ -0,0 +1,10 @@
+#include <string.h>
+
+int strcmp(const char* str1, const char* str2) {
+    while (*str1 != '\0') {
+        if (*str1 != *str2) return (int)*str1 - (int)*str2;
```

Comment:
You can eliminate this if statement and move the condition into the while statement

Reply:
I don't think the compiler makes any difference, but it improves readability maybe :)